src/test/java/org/openrewrite/staticanalysis/EqualsAvoidsNullVisitorTest.java

```diff
@@ -23,7 +23,7 @@
 import static org.openrewrite.java.Assertions.java;
 
 @SuppressWarnings({"ClassInitializerMayBeStatic", "StatementWithEmptyBody", "ConstantConditions"})
-class EqualsAvoidsNullTest implements RewriteTest {
+class EqualsAvoidsNullVisitorTest implements RewriteTest {
```

Comment:
On line 30 we're testing the recipe though right? Perhaps best then to not change the class name.
```suggestion
class EqualsAvoidsNullTest implements RewriteTest {
```

Reply:
Yes, but previously it was kind of broken because the test auto-discovery couldn’t function due to the missing naming convention. Now it’s working in every direction. I had to run the build to find the tests, so please avoid this pitfall for the next person.


<img width="763" alt="image" src="https://github.com/user-attachments/assets/7bb46180-da40-43b7-99de-7f692daaa0bd">

<img width="703" alt="image" src="https://github.com/user-attachments/assets/f3528634-e72f-4660-adea-8ec14fe5c75d">